src/index.js

```diff
@@ -1,3 +1,7 @@
+//Utility polyfills
+import './util/object-assign';
```

Comment:
Are the new imports being used?

Reply:
`Object.assign()` just once, `Array.find()` everywhere.